apps/sync/test/unit/sync-engine/syncengine_test.js

```diff
@@ -9,6 +9,7 @@
   FxSyncWebCrypto,
   Kinto,
   requireApp,
+  setup,
```

Comment:
I don't know why we need to declare mocha related globals here. This is not happening with other apps. Could you file a bug to investigate it, please? Thanks!